tsconfig.json

```diff
@@ -39,6 +39,9 @@
       "@spartacus/my-account/organization/assets": [
         "feature-libs/my-account/organization/assets/public_api"
       ],
+      "@spartacus/my-account/organization/root": [
```

Comment:
It should be added in different tsconfigs as well. SSR is now broken, because we don't have any @spartacus/my-account paths configured there.